setup.py

```diff
@@ -14,8 +14,8 @@
     license="MIT",
     install_requires=["requests"],
     include_package_data=True,
-    package_dir={'razorpay': 'razorpay', 'razorpay.resources': 'razorpay/resources'},
-    packages=['razorpay', 'razorpay.resources'],
+    package_dir={'razorpay': 'razorpay', 'razorpay.resources': 'razorpay/resources', 'razorpay.constants': 'razorpay/constants', 'razorpay.utility': 'razorpay/utility'},
+    packages=['razorpay', 'razorpay.resources', 'razorpay/constants', 'razorpay/utility'],
```

Comment:
Can you elaborate why these should be added here?

Reply:
We have to show what directories are included with razorpay-python-master.zip package in setup.py file to say that the installing folders belongs to razorpay and they come with package
---------------------
 constants and utility folders of razorpay folder is not entered in setup.py file package directory  and packages line but they are in razorpay-python-master.zip package, so the warning attached in image appears
---------------------
**before this edit** during `python setup.py install` , check the warning
![Screenshot (7)](https://user-images.githubusercontent.com/112264612/187820055-79da3e06-d051-4b5f-b8c6-45bf66772917.png)
**after this edit** during `python setup.py install` , the warning disappears
![Screenshot (10)](https://user-images.githubusercontent.com/112264612/187820432-10bfdb2e-7a19-4227-aedc-d4ecbfd6ae30.png)
----------------------

Reply:
> Can you elaborate why these should be added here?

small change in 18th line and I have updated it, check it
placed `razorpay/constants` in the place of `razorpay.constants` and `razorpay/utility` in the place of `razorpay.utility`
the previously entered line is `packages=['razorpay', 'razorpay.resources', 'razorpay/constants', 'razorpay/utility'],` and corrected 18th line (packages definition line) is
`packages=['razorpay', 'razorpay.resources', 'razorpay.constants', 'razorpay.utility'],`

Reply:
Thanks! Will review it and merge it soon!
Thank you for the contribution! 😄

Reply:
thank you